docs/email/postfix/configure-spf-and-dkim-in-postfix-on-debian-8.md

```diff
@@ -413,3 +434,5 @@ The reason the YYYYMM format is used for the selector is that best practice call
     Make sure they both start without any errors.
 
 7.  After a couple of weeks, all email in transit should either have been delivered or bounced and the old DKIM key information in DNS won't be needed anymore. Delete the old `YYYYMM._domainkey` TXT records in each of your domains, leaving just the newest ones (most recent year and month). Don't worry if you forget and leave the old keys around longer than planned. There's no security issue. Removing the obsolete records is more a matter of keeping things neat and tidy than anything else.
+
+##
```

Comment:
@tknarr Thanks for clarifying. I've removed the OpenDMARC references from the beginning of the doc for now, just to prevent any confusion. I've also added a breakdown / explanation of the different tags used in the DMARC examples. Could you take a look and merge the changes?